release-content/migration-guides/wgpu_25.md

```diff
@@ -0,0 +1,22 @@
+---
+title: `wgpu` 25
+pull_requests: [ 19563 ]
+---
+
+`wgpu` 25 introduces a number of breaking changes, most notably in the way Bevy is required to handle
+uniforms with dynamic offsets which are used pervasively in the renderer. Dynamic offsets and uniforms
+of any kind are no longer allowed to be used in the same bind group as binding arrays. As such, the
+following changes to the default bind group numbering have been made in 3d:
+
+- `@group(0)` view uniforms
+- `@group(1)` view uniforms requiring binding arrays
+- `@group(2)` mesh uniforms
+- `@group(3)` material uniforms
```

Comment:
If we're telling users to use group 3 for their material uniforms, doesn't that mean that we can't put our bindless stuff in group 3? (eg. crates/bevy_render/src/bindless.wgsl or assets/shaders/bindless_material.wgsl)